Source/UI/NewUI/BillingForm/View/BillingFormPhoneNumberText.swift

```diff
@@ -3,32 +3,41 @@ import PhoneNumberKit
 
 protocol BillingFormPhoneNumberTextDelegate {
     func updateCountryCode(code:Int)
+    func isInValidNumber()
 }
 
 final class BillingFormPhoneNumberText: BillingFormTextField {
+
+    // MARK: - Properties
+
     var phoneNumberTextDelegate: BillingFormPhoneNumberTextDelegate?
     var phoneNumber: PhoneNumber?
+    let phoneNumberKit = PhoneNumberKit()
 
+    var partialFormatter: PartialFormatter {
+       PartialFormatter(phoneNumberKit: phoneNumberKit, defaultRegion: "GB", withPrefix: true)
+    }
+
+    /// National Number (e.g. )
     var nationalNumber: String {
-        let rawNumber = self.text ?? ""
+        let rawNumber = text ?? ""
         return partialFormatter.nationalNumber(from: rawNumber)
     }
 
+    /// True if the phone number is valid, false otherwise
```

Comment:
do we need this comment ? the isValidNumber is clear enough what it does ? If i am correct

Reply:
please suggest code using <cmd+g>